tasks/main.yml

```
@@ -40,6 +40,7 @@
     group: root
   with_items:
     - "{{ etcd_certificates }}"
+  no_log: true
```

```suggestion
  no_log: '{{ ansible_verbosity < 3 }}'
```
Maybe the user should have the possibility to see the output by setting the verbosity accordingly.

---

Sure, that's fine by me. The important thing in my opinion is that it does not output the private key by default.